src/main/java/com/example/bank/registration/jpa/Client.java

```diff
@@ -0,0 +1,39 @@
+package com.example.bank.registration.jpa;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Entity(name = "client")
+@Table(name = "client")
```

Comment:
W tym PR juz tego nie rob ale na za tydzien misje masz taka.
Przejsc po kazdej ze swoich encji i masz za zadanie stwierdzic jak przydatna jest w Twoim konkretnym przypadku dana adnotacja. 
Tu np jak masz table i name client to jest bardzo duza szansa ze wlasnie tak samo nazwana tabela sie zrobi i bez jej uzycia :) 
Duzo z tych adnotacji jak pokazywalem to pokazywalem mozliwosci ich. Ale jezeli uzywasz np domyslnych wartosci w adnotacjach niektorych to nie powinno tam ich byc w ogole w sensie tych wartosci. 
Na przyklad w @Column nullable = true to jest domyslna wartosc. Tu nigdzie nie masz tego tak uzytego i bardzo dobrze lae chodzi mi o wszystkie adnotacje od JPA ktorych uzywasz

Reply:
Tak, już kiedyś nawet o to pytałem w komentarzu lub na spotkaniu. Przy kolejnym commicie to zrobię.